arches/app/tasks.py

```diff
@@ -208,29 +208,38 @@ def on_chord_error(request, exc, traceback):
 def load_branch_csv(userid, files, summary, result, temp_dir, loadid):
     from arches.app.etl_modules import branch_csv_importer
 
-    BranchCsvImporter = branch_csv_importer.BranchCsvImporter(request=None, loadid=loadid, temp_dir=temp_dir)
-    BranchCsvImporter.run_load_task(files, summary, result, temp_dir, loadid)
+    try:
+        BranchCsvImporter = branch_csv_importer.BranchCsvImporter(request=None, loadid=loadid, temp_dir=temp_dir)
+        BranchCsvImporter.run_load_task(files, summary, result, temp_dir, loadid)
 
-    load_event = models.LoadEvent.objects.get(loadid=loadid)
-    status = _("Compeleted") if load_event.status == "indexed" else _("Failed")
-    msg = _("Branch Excel Import: {} [{}]").format(summary["name"], status)
-    user = User.objects.get(id=userid)
-    notify_completion(msg, user)
+        load_event = models.LoadEvent.objects.get(loadid=loadid)
+        status = _("Completed") if load_event.status == "indexed" else _("Failed")
+        msg = _("Branch Excel Import: {} [{}]").format(summary["name"], status)
+        user = User.objects.get(id=userid)
+        notify_completion(msg, user)
+    except:
```

Comment:
Could you modify this (and the other except statements in this pr) so that the error gets logged? 
```
except Exception as e:
    logger.error(e)
```